src/devices/Bmxx80/Bme680HeaterProfile.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Bmxx80
+{
+    /// <summary>
+    /// 10 addressable heater profiles stored on the Bme680.
+    /// </summary>
+    public enum Bme680HeaterProfile : byte
+    {
+        /// <summary>
+        /// Heater Profile 1.
```

Comment:
Is there any other info on what does each profile mean in the datasheet? Given that this enum is public, I think it might be confusing to know which one to pick. Unless consumers would likely understand what each profile mean.

Reply:
These heater profiles are perfectly equal, they have no inherent settings associated with them. All profiles must be configured by the user through `ConfigureHeatingProfile`. It doesn't matter which profile is used.

Reply:
To make this a little clearer, there are in total 10x2 registers for these settings.
So for each profile a target temperature and duration for the heater. An additional register decides which of these profiles is used (by writing one of the values of the enum to it).